src/.#Katla.idr

```diff
@@ -0,0 +1 @@
+jfdm@sleipnir.11322:1620120778
```

Comment:
Probably did not mean to include this file

Reply:
urgh